view/frontend/web/template/retailer/search.html

```diff
@@ -46,26 +53,61 @@
     </div>
 
     <div class="shop-details" data-bind="{scope: requestChild('store-list'), visible: selectedMarker() !== null }">
-        <button type="button"
-                class="action close"
-                data-action="close"
-                data-bind="{ click : $parent.resetBounds.bind($parent), attr: { title: $t('Close') }}">
-            <span data-bind="i18n: 'Close'"></span>
-        </button>
+
         <!-- ko if: ($parent.selectedMarker() && $parent.selectedMarker.bind($parent)) -->
             <div class="shop-details-wrapper" data-bind="{template: { name: storeDetailTemplate, data: $parent.selectedMarker() }}"></div>
         <!-- /ko -->
-        <!-- ko if: $parent.nearbyMarkers -->
-        <div class="search-result-list" data-bind="{ visible: $parent.nearbyMarkers().length > 0}">
-            <div class="search-result-header">
-                <p data-bind="text: $t(nearbySearchLabel).replace('%s', $parent.countNearbyMarkers.bind($parent))"></p>
-            </div>
-            <ul data-bind="foreach: $parent.nearbyMarkers">
-                <li data-bind="template: $parent.storeListTemplate" class="result-item"></li>
-            </ul>
+        <!--ko if: $parent.nearbyMarkers -->
+        <!--<div class="search-result-list" data-bind="{ visible: $parent.nearbyMarkers().length > 0}">-->
+            <!--<div class="search-result-header">-->
+                <!--<p data-bind="text: $t(nearbySearchLabel).replace('%s', $parent.countNearbyMarkers.bind($parent))"></p>-->
+            <!--</div>-->
+            <!--<ul data-bind="foreach: $parent.nearbyMarkers">-->
+                <!--<li data-bind="template: $parent.storeListTemplate" class="result-item"></li>-->
+            <!--</ul>-->
+        <!--</div>-->
+         <!--/ko -->
+        <div class="close-btn-wrapper">
+            <button type="button"
+                    class="action close"
+                    data-action="close"
+                    data-bind="{ click : $parent.closeDetails.bind($parent), attr: { title: $t('Close') }}">
+                <span data-bind="i18n: 'Close'"></span>
+            </button>
         </div>
-        <!-- /ko -->
     </div>
 </div>
 
 <div class="map" data-bind="afterRender: initMap"> </div>
+
+<div class="store-locator-services-container" data-bind="{scope: requestChild('service'), visible: selectedMarker() !== null}">
```

Comment:
Storelocator must stay independant of the modules, you  must overload this html in service module or try a solution to add this html after the storelocator code in the page